homepage/migrations/0006_review_test_data.py

```diff
@@ -0,0 +1,50 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('homepage', '0001_initial'),
+        ('homepage', '0002_user_test_data'),
+        ('homepage', '0003_course_test_data'),
+        ('homepage', '0005_professor_test_data')
+    ]
+
+    def generate_data(apps, schema_editor):
+        from homepage.models import AppUser
+        from homepage.models import Course
+        from homepage.models import Review
+        from homepage.models import Professor
+        from django.utils import timezone
+        from django.core.files.uploadedfile import SimpleUploadedFile
+        from django.conf import settings
+
+        users = AppUser.objects.all()
+        courses = Course.objects.all()
+        professors = Professor.objects.all()
+
+        image_path = settings.MEDIA_ROOT + '/images/test_image.jpg'
+        image = SimpleUploadedFile(
+            name='new_test_image.jpg',
+            content=open(image_path, 'rb').read(),
+            content_type='image/jpeg')
+
+        test_data_review = [
+            (1, courses[0].course_id, users[0].user.id, 5, timezone.now(),
+                "Great course", 3, 10, professors[2].id, image),
+            (2, courses[0].course_id, users[0].user.id, 4, timezone.now(), "I've learned a lot!", 2, 4),
+            (3, courses[1].course_id, users[1].user.id, 4.5, timezone.now(),
```

Comment:
what order is the input in? since in line 35 there are 2 integers after the string, and in line 38 there is one floating  point number after it.

Reply:
I added "number of likes" to the two first rows - 10 likes, and 4.
The first number was there - it's the rating. 
That's according to the original order of the model fields